files/runden2.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+
+enum Runden{
+	mathe,
+	ab,
+	auf
+};
+
+int runden(double d, Runden mode){
+	 int i = d; //speichert Double in Integer, schneidet Kommastellen ab
+    if(mode == mathe){
+       if(d - i < 0.5){
+		     return i;
+		 }
+		 else {
+			 return i + 1;
+		 }
+	 }
+	 else if (mode == ab){
+		 return i;
```

Comment:
bei zahlen kleiner 0 wird da aufgerundet könnte man als Aufgabe in den Spiel teil setzen das korrekt zu machen